cli/list/list_test.go

```diff
@@ -0,0 +1,29 @@
+package list
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSort(t *testing.T) {
+	type argsData struct {
+		input, expected []string
+	}
+
+	testArgsData := []argsData{
+		{[]string{"0.3.0", "1.3.0", "0.1.0"},
```

Comment:
A user can create any directories in bin-dir. Please add tests for invalid versions: string instead of int64, different versions length.

Reply:
added more test cases.